pkg/jobs/canary/canary_jobs.go

```diff
@@ -415,7 +415,7 @@ func ReconcileDeletedCanaryChecks() {
 		ID        string
```

Comment:
@yashmehrotra, do we still need this job since we have this now
https://github.com/flanksource/duty/pull/206/files